homeassistant/components/griddy/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "title": "Griddy",
+    "step": {
+      "user": {
+        "title": "Setup your Griddy Load Zone",
+		"description" : "Your Load Zone is in your Griddy account under “Account > Meter > Load Zone.”",
+        "data": {
+          "loadzone": "Load Zone (Settlement Point)"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Failed to connect, please try again",
+      "invalid_auth": "Invalid authentication",
+      "unknown": "Unexpected error"
+    },
+    "abort": {
+      "already_configured": "Griddy is already configured"
```

Comment:
```suggestion
      "already_configured": "This load zone is already configured"
```

Reply:
Extra: should we filter out zone point that have already been configured from the config flow?

Reply:
Or: should we use a multi select to allow users to pick the load zones they want to track ?

Reply:
I don't think there is much use case to track multiple as it would be exceedingly rare that they would have service in multiple zones.

Reply:
Adjusted the message